### DISCUSSION TITLE
Dropbox

### DISCUSSION THREAD

**Discussion by Petahwil** [jrnl-org/jrnl#1396] [Q&A] [answered]
I am having trouble finding any info on how to link a jrnl with Dropbox so that I can use it on multiple computers. I also have an existing jrnl that i want to move to dropbox. Any info would help thanks!

**Comment by giuliofoletto** [comment, marked as answer]
I think you can move the journal file to the dropbox folder in your computer, so that it gets synced. To do this:

1.  Move the journal text file from its location to the dropbox folder (using `mv` or your file explorer).
2.  Change the path to the journal file in the jrnl configuration yaml file (located in `~/.config/jrnl/jrnl.yaml`, where `~` is your home or userprofile folder). The entry to modify is `journals: default:`.
